boards/arm/adafruit_grand_central_m4_express/doc/index.rst

```diff
@@ -0,0 +1,194 @@
+.. _adafruit_grand_central_m4_express:
```

Comment:
You are adding a new board. In this case you should squaseh documentation in the previous commit.

Reply:
Ping